.github/workflows/test.yml

```diff
@@ -43,3 +44,95 @@ jobs:
           ATLAS_PUBLIC_KEY: ${{ secrets.ATLAS_PUBLIC_KEY }}
           ATLAS_PRIVATE_KEY: ${{ secrets.ATLAS_PRIVATE_KEY }}
 
+  prepare-e2e:
+    name: Prepare E2E configuration and image
+    needs: [unit-test, int-test]
```

Comment:
can we please remove these and just run the test in parallel with the others?

Reply:
agree with int-test as they take 20 min. I think it makes sense to wait for unit-test as it take a min, but will avoid an unnecessary run of e2e in case of error.